packages/form-footer/src/FormFooter.tsx

```diff
@@ -20,97 +23,90 @@ import PrimaryButton, { PrimaryButtonProps } from './PrimaryButton';
-              leftGlyph={_cancelButtonProps.leftGlyph}
-              darkMode={darkMode}
-              data-testid="lg-form_footer-cancel_button"
+              variant={ButtonVariant.Default}
+              leftGlyph={
+                <ArrowLeftIcon data-testid={LGIDS_FORM_FOOTER.backButtonIcon} />
+              }
+              data-testid={LGIDS_FORM_FOOTER.backButton}
+              {...backButtonProps}
             >
-              {_cancelButtonProps.children || 'Cancel'}
+              {backButtonProps?.children || 'Back'}
             </Button>
           )}
-          {isComponentType(primaryButton as React.ReactElement, 'Button') ? (
-            React.cloneElement(primaryButton as React.ReactElement, {
-              darkMode: darkMode,
-              ['data-testid']: 'lg-form_footer-primary_button',
-            })
-          ) : (
-            <PrimaryButton
-              darkMode={darkMode}
-              data-testid="lg-form_footer-primary_button"
-              {...(primaryButton as PrimaryButtonProps)}
-            />
-          )}
+          <div className={flexEndContent}>
+            {errorMessage && (
+              <Banner className={bannerStyle} variant={BannerVariant.Danger}>
+                {errorMessage}
+              </Banner>
+            )}
+            {showCancelButton && (
+              <Button
+                data-testid={LGIDS_FORM_FOOTER.cancelButton}
+                {...cancelButtonProps}
+                variant={ButtonVariant.Default}
+              >
+                {cancelButtonProps?.children || 'Cancel'}
+              </Button>
+            )}
+            {showDeprecatedPrimaryButton ? (
+              isComponentType(primaryButton as React.ReactElement, 'Button') ? (
+                React.cloneElement(primaryButton as React.ReactElement, {
+                  ['data-testid']: LGIDS_FORM_FOOTER.primaryButton,
+                })
+              ) : (
+                <PrimaryButton
+                  data-testid={LGIDS_FORM_FOOTER.primaryButton}
+                  {...(primaryButton as PrimaryButtonProps)}
+                />
+              )
+            ) : (
+              <Button
+                variant={ButtonVariant.Primary}
+                data-testid={LGIDS_FORM_FOOTER.primaryButton}
+                {...primaryButtonProps}
+              />
+            )}
+          </div>
         </div>
-      </div>
-    </footer>
+      </footer>
+    </LeafyGreenProvider>
   );
 }
+
+FormFooter.displayName = 'ConfirmationModal';
+
+FormFooter.propTypes = {
+  contentClassName: PropTypes.string,
+  errorMessage: PropTypes.string,
+  darkMode: PropTypes.bool,
+  // eslint-disable-next-line react/forbid-prop-types
+  primaryButtonProps: PropTypes.objectOf(PropTypes.any),
+  // eslint-disable-next-line react/forbid-prop-types
+  cancelButtonProps: PropTypes.objectOf(PropTypes.any),
+  // eslint-disable-next-line react/forbid-prop-types
+  backButtonProps: PropTypes.objectOf(PropTypes.any),
```

Comment:
same note as above about intentionally leaving these disables out for future cleanup